lib/app.js

```diff
@@ -61,6 +62,13 @@ app.post('/cloud/:datasetId', function(req, res) {
 var excludedPaths = ['/authpolicy', '/file'];
 app.use(require('fh-wfm-user/lib/middleware/validateSession')(mediator, mbaasApi,excludedPaths));
 
+
+//Initialising the fh-wfm-sync data sets for each module
+wfmSync.init(mediator, mbaasApi, 'workorders', config.get("syncOptions"));
+wfmSync.init(mediator, mbaasApi, 'result', config.get("syncOptions"));
+wfmSync.init(mediator, mbaasApi, 'workflows', config.get("syncOptions"));
+wfmSync.init(mediator, mbaasApi, 'messages', config.get("syncOptions"));
```

Comment:
Yes, the applications should be responsible for setting up their data sets. 

The principle is that the raincatcher-sync module is just one implementation of the sync mediator topics. We don't want to bake that into the other modules by including the raincatcher-sync module in the other modules.

Also, it allows developers to set up raincatcher-sync data sets for their own custom data sets.